releases/0.4.0/index.md

```diff
@@ -4,16 +4,58 @@ layout: releases
 custom_css: releases
 ---
 
-## {{ page.title }}
+# {{ page.title }}
 
-###### Date: _TBD_
+##### Date: 2021-05-27
 
-#### Ontology File(s)
+## Ontology File(s)
 
-[GitHub](https://github.com/casework/CASE/tree/develop)
+[GitHub](https://github.com/casework/CASE/releases/tag/0.4.0)
 
-#### Migration guidance
+## Migration guidance
 
 A guide to migration from CASE 0.3.0 will be provided.
 
-#### Release Notes
+## Release Notes
+
+### Focus
+
+The focus of this release is on providing an easy way to produce a provenance chain for evidence objects up to and including their moment of emergence in the investigation. This is modeled after the [PROV-O ontology](https://www.w3.org/TR/prov-o/).
+
+### Changes
+
+*(These are changes to ontologies, classes or properties in the preexisting ontology.)*
+
+Throughout this section, many of the notes are tied to Change Proposals.  These proposals are available on CASE's [Reference Material page](/resources/references.html).
+
+#### Development Changes
+
+*(These are changes to ontology development practice, and are not necessarily changes to ontology data.)*
+
+* Revise merge review procedures and ChangeLog format to allow asynchronous change proposal posting
+  *(Change Proposal 27)* 
+
+#### Breaking Changes
+
+*(These are changes to ontologies, classes or properties in the preexisting ontology that make the new release non-backward-compatible.)*
+
+* None
+
+
+#### Changes
+
+*(These are general changes to the preexisting ontology that are not breaking or range changes.)*
+
+* Revise ontology IRI to be slash-based and drop resource-type
+  *(Change Proposal 34
```

Comment:
This is a minor syntax error, which I'm guessing will be caught when the link is populated.

Reply:
This was fixed in [PR 107](https://github.com/casework/casework.github.io/pull/107).